app/tests/test_parcel.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+import os
+import json
+
+
+from app import create_app
+
+
+class ParcelTestCase(unittest.TestCase):
+    """This test case tets the parcel test cases"""
+    def setUp(self):
+        """Define test variables and initialize app."""
+        self.app = create_app(config_name="testing")
+        self.client = self.app.test_client
+        self.parcel = {"user_id": 1,
+                       "parcel_type": "letter",
+                       "recepient_number": "254715428709",
+                       "Dest": "Moi_avenue",
+                       "status": "on_transit"
+                       }
+
+    def test_user_add_parcel(self):
+        """Test API if it adds a parcel(POST)"""
+        res = self.client().post("api/v1/parcels", json=self.parcel)
+        self.assertEqual(res.status_code, 201)
+        self.assertIn("order_no", str(res.data))
+
+    def test_user_add_parcel_bad_Request(self):
+        """Test API for bad request"""
+        parcel = {"user_id": 1,
+                  "parcel_type": "letter",
+                  "recepient_number": "254715428709",
+                  "status": "on_transit"
+                  }
+        res = self.client().post("api/v1/parcels", json=parcel)
+        self.assertEqual(res.status_code, 400)
+        self.assertIn("destination", str(res.data))
+
+    def test_admin_all_parcels(self):
```

Comment:
Rename this function name to reflect what it tests. It does not have anything to do with admin